### ISSUE TITLE
Local flags broken for Overwatch

### ISSUE THREAD

**Issue by Resike** [WoW-Tools/CASCExplorer#11]
Selecting a different locale fro Overwatch always return all the files, because every file has "All Locale" flag for some reason.

**Comment by tomrus88** [comment]
I don't think this is true, it only loads (or should load) enUS files at the moment and it can't be changed from UI despite selection is available.

**Comment by Resike** [comment]
It loads every file, thats for sure. How come an older version of the program sees different locale flags and loads them properly?
http://i.imgur.com/DIkAYWW.png

**Comment by Resike** [comment]
Also the (X names missing) counter seems to be always 0 when you change locales.
